pkg/apis/core/types.go

```diff
@@ -2430,6 +2430,14 @@ const (
 	PodReasonUnschedulable = "Unschedulable"
 	// ContainersReady indicates whether all containers in the pod are ready.
 	ContainersReady PodConditionType = "ContainersReady"
+	// PreemptedByScheduler indicates that the Pod was preempted
+	PreemptedByScheduler PodConditionType = "PreemptedByScheduler"
+	// EvictedByTaints indicates that the Pod was evicted due to taints
+	EvictedByTaints PodConditionType = "EvictedByTaints"
+	// DeletedFromUnusableNode indicates that the Pod was deleted from an unusable node
+	DeletedFromUnusableNode PodConditionType = "DeletedFromUnusableNode"
+	// EvictedByAPI indicates that the Pod was evicted due to an API-initiated eviction.
+	EvictedByAPI PodConditionType = "EvictedByAPI"
```

Comment:
@alculquicondor is there a documented process for reviewing new condition types . This seems like needing an api review

Reply:
It will, indeed. We'll request one once it's past draft.

Beyond API review, there's no other process I'm aware of.